evennia/commands/default/building.py

```diff
@@ -2548,6 +2548,10 @@ def format_email(self, obj):
         if hasattr(obj, "email") and obj.email:
             return f"{self.detail_color}{obj.email}|n"
 
+    def format_last_login(self, obj):
+        if hasattr(obj, "last_login") and obj.last_login:
+            return f"{self.detail_color}{obj.last_login}"
```

Comment:
For information, you are missing a `|n` after this return, without it, the color will bleed. I'll add that before merging.